_licenses/lal-1.3.txt

```diff
@@ -0,0 +1,214 @@
+---
+title: Licence Art Libre 1.3
+spdx-id: LAL-1.3
+
+description: This license is written in consideration of both French law and the Berne Convention for the Protection of Literary and Artistic Works. Frequently used for media assets such as logos. When incorporated into larger works, the larger work is expected to be either publicly accessable or similarly licensed.
+
+how: Create a text file (typically named LICENSE or LICENSE.txt) in the root of your source code and copy the text of the license into the file. It is also acceptable to solely supply a link to a copy of the license, usually to the <a href='https://artlibre.org/'>canonical URL for the license</a>.
+
+using:
+  - Open Source Publishing: https://github.com/osp/osp.workshop.flatland/blob/master/licences/lal.txt
+  - linuxfr.org: https://github.com/mjourdan/linuxfr-design/blob/master/LICENCE.md
+  - Oak: https://github.com/Debetux/Oak/blob/master/template/default/LICENCE
+
+permissions:
+  - commercial-use
+  - modifications
+  - distribution
+  - private-use
+
+conditions:
+  - include-copyright
+  - document-changes
+  - same-license
+
+limitations:
+  - liability
```

Comment:
I'm not sure about this. The description of the field is "This license includes a limitation of liability", and most licenses are very explicit about that.

LAL-1.3 only has (unless I'm missing something), quoting the English version:

> 7. YOUR RESPONSIBILITIES 
The freedom to use the work as defined by the Free Art License (right to copy, distribute, modify) implies that everyone is responsible for their own actions.

Reply:
§7 was my rationale for leaving the liability flag. @creativecommons evaluated the English translation of LAL-1.3 found it and CC-BY-SA-4.0 compatible. They have comments at https://wiki.creativecommons.org/wiki/ShareAlike_compatibility_analysis:_FAL and https://wiki.creativecommons.org/wiki/ShareAlike_compatibility:_FAL but neither mentions liability.

Open Knowledge and tldrlegal at https://opendefinition.org/licenses/fal/ and https://tldrlegal.com/license/free-art-license-(fal) both use the English version but neither provide summaries yet.

A writer in the European Journal of Law and Technology emailed the drafters for clarification on §7, described at http://ejlt.org/article/view/237/410#_ftnref18 and reports that "The intention of the drafters [18] is to recall to the licensors that they cannot license works on which they do not hold the rights."

Reply:
Thanks for those references! I'd read them all before and completely forgotten about each of them. 😄 It seems unclear at best. I'd prefer to remove the field -- the limitations group should look like this...

https://github.com/github/choosealicense.com/blob/518c4bcf7a8750dc7ebb3612b79730c4674c8f61/_licenses/wtfpl.txt#L19

...the field can always be added back if more info surfaces that justifies it.

Reply:
Sounds good! Liability is removed.